### ISSUE TITLE
Attachment

### ISSUE THREAD

**Issue by AppoPro** [jonmbake/bootstrap3-contact-form#13]
How about to add support of attachments ?

**Comment by jonmbake** [comment]
Yes, this is possible.  Sounds like a good idea.  I'll try to get to it sometime this week.

**Comment by pixilus** [comment]
First thanks for your great contact form but could you please add attachments? :)

I've tried to implement it myself but nothing works.
